annotations/transport/annotations_router.go

```diff
@@ -252,16 +251,11 @@ func storedAnnotationsToReadAnnotations(s []influxdb.StoredAnnotation) (influxdb
 	r := influxdb.ReadAnnotations{}
 
 	for _, val := range s {
-		stickers, err := stickerSliceToMap(val.Stickers)
```

Comment:
Using the same type for stickers throughout the application code makes conversions like this not needed anymore